build/release/teamcity-compile-and-publish-prerelease-build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
make sure the permissions for these are `755`

Reply:
The are now `755`. Originally they were `744`.